vortex-io/src/dispatcher/mod.rs

```diff
@@ -73,7 +75,7 @@ impl Default for IoDispatcher {
         #[cfg(all(feature = "compio", not(feature = "tokio")))]
         return Self(Inner::Compio(CompioDispatcher::new(1)));
         #[cfg(not(any(feature = "compio", feature = "tokio")))]
-        return Self(Inner {});
+        vortex_panic!("must enable one of compio or tokio to use IoDispatcher");
```

Comment:
If we wrap this puppy in a `const {}` block it would fail at compile time instead of runtime, just a thought 🤷

Reply:
<img width="1504" alt="image" src="https://github.com/user-attachments/assets/4c959dd6-c198-46e6-a7f9-2c59ea5543ff">

VS

<img width="1481" alt="image" src="https://github.com/user-attachments/assets/5e63916e-abce-43fb-8726-e198ab4dbfd1">